docs/src/main/paradox/bootstrap/index.md

```diff
@@ -28,15 +28,16 @@ A discovery mechanism needs to be chosen. A good default choice is DNS.
 
 Add `akka-management-cluster-bootstrap` and one or more discovery mechanisms to use for the discovery process.
 
-For example, you might choose to use the DNS discovery and bootstrap extensions:
+For example, you might choose to use the [DNS discovery](https://doc.akka.io/docs/akka/current/discovery/index.html#discovery-method-dns)
+and bootstrap extensions:
 
 @@dependency[sbt,Gradle,Maven] {
   group=com.lightbend.akka.management
   artifact=akka-management-cluster-bootstrap_$scala.binary_version$
   version=$version$
-  group2=com.lightbend.akka.discovery
-  artifact2=akka-discovery-dns_$scala.binary_version$
-  version2=$version$
+  group2=com.typesafe.akka
+  artifact2=akka-discovery_$scala.binary_version$
+  version2=$akka.version$
```

Comment:
It isn't strictly necessary to include a direct dependency on `akka-discovery`, since `akka-management-cluster-bootstrap` already depends on it. If you think it's better to keep it for clarity sake or to ensure version alignment between core Akka modules, that's OK but I wanted to check.

Reply:
right, I didn't think of that it's already a transitive dependency. We might want to keep it explicit here for clarity and I think this is a high risk of being the wrong version if included via transitive, e.g. Akka Management will typically be built with an older version (2.5.20) and the application is using a newer (2.5.21). This will show the version that Akka management was built with but then it's at least easy to see what to override.

I'll add a note that the Akka version here should be the same as for other Akka dependencies included by the application, which also must be same or higher than what is shown here.

Reply:
added the note: ce24b5f